Neos.ContentRepository/Classes/Security/Authorization/Privilege/Node/AbstractNodePrivilege.php

```diff
@@ -63,19 +63,9 @@ public function initialize()
             return;
         }
         $this->initialized = true;
-
+        $this->eelCompilingEvaluator = $this->objectManager->get(CompilingEvaluator::class);
         $this->nodeContext = new $this->nodeContextClassName();
-        $eelContext = new Context($this->nodeContext);
-
-        $this->eelCompilingEvaluator = new CompilingEvaluator();
-
-        $this->eelCompilingEvaluator->evaluate($this->getParsedMatcher(), $eelContext);
-
-        $methodPrivilegeMatcher = $this->buildMethodPrivilegeMatcher();
-
-        $methodPrivilegeTarget = new PrivilegeTarget($this->privilegeTarget->getIdentifier() . '__methodPrivilege', MethodPrivilege::class, $methodPrivilegeMatcher);
-        $methodPrivilegeTarget->injectObjectManager($this->objectManager);
-        $this->methodPrivilege = $methodPrivilegeTarget->createPrivilege($this->getPermission(), $this->getParameters());
+        $this->initializeMethodPrivilege();
     }
```

Comment:
this method is now only used in `CreateNodePrivilege`, might as well just move the creation of the evaluator there

Reply:
I think due to the latest changes this comment is no longer valid. I will merge now with the votes we have.